client/src/components/Swap/SwapModal.tsx

```diff
@@ -135,16 +142,36 @@ export const SwapModal: React.FC<SwapModalProps> = ({
 
         {paymentPlatform !== PaymentPlatform.GARANTI ? (
           <div>
-            <QRContainer>
-              <QRCode
-                value={link}
-                size={192}/>
-            </QRContainer>
-            <QRLabel>
-              <Link href={troubleScanningQRCodeLink} target="_blank">  
-                {paymentPlatform === PaymentPlatform.REVOLUT ? `Send via Revolut.com ↗` : "Trouble scanning QR?"}
-              </Link>
-            </QRLabel>     
+            {
+              isMobile ? (
+                <PaymentButtonContainer>
+                  <ButtonContainer>
+                    <Button
+                      onClick={() => {
+                        setPaymentInitiated(true);
+                        window.open(link, '_blank', 'noopener,noreferrer');
+                      }}
+                    >
+                      {`Send ₹${amount} on ${paymentPlatformName} ↗`}
+                    </Button> 
+                  </ButtonContainer>
+                </PaymentButtonContainer>
+              ) :
+              (
+                <>
+                  <QRContainer>
```

Comment:
We need to do more work here to spec out / add in the "Desktop Only" modal for Revolut before launching this.